game-core/build.gradle

```diff
@@ -121,6 +121,18 @@ dependencies {
     testCompile 'org.junit.jupiter:junit-jupiter-api:5.0.2'
     testCompile 'org.mockito:mockito-core:2.13.0'
     testCompile 'org.sonatype.goodies:goodies-prefs:2.2.4'
+    testCompile 'org.testfx:testfx-junit5:4.0.12-alpha'
+
+    if (JavaVersion.current() == JavaVersion.VERSION_1_9) {
```

Comment:
I'll admit this makes me nervous. I'm less and less sure about why it's helpful to build java9; seems we are making things more complicated. This block is only necessary so the 1.9 build passes, right?

Reply:
> I'm less and less sure about why it's helpful to build java9

While I was looking at the Java 9 build logs, I noticed there are several strange messages being logged while running the unit tests that do not occur in the Java 8 build:

```
Warning: serialization de-serializatoin error, m_name in DefaultNamed.java is null.
```

So, theoretically, one benefit of the Java 9 build is it gives us a heads-up on migration issues.

Reply:
The heads-up is nice, I agree. Though, 'theoretical' and 'nice' seem to be the operative words :|